src/main/java/ru/practicum/shareit/user/UserServiceImpl.java

```diff
@@ -5,49 +5,68 @@
 import org.springframework.stereotype.Service;
 import ru.practicum.shareit.exceptions.NotFoundUserException;
 import ru.practicum.shareit.user.dto.UserDto;
+import ru.practicum.shareit.user.model.User;
 
-import java.util.Collection;
+import java.util.List;
+
+import static ru.practicum.shareit.user.UserMapper.toUser;
+
+/**
+ * User service impl.
+ */
 
 @Service
 @Slf4j
 public class UserServiceImpl implements UserService {
-    private final UserStorage userStorage;
+    private final UserRepository userRepository;
+
+    public static void checkUserExistsById(UserRepository userRepository, Long userId) {
```

Comment:
этот метод точно должен быть публичным? А если да, то не должен ли он оказаться в интерфейсе? Твоё мнение?

Reply:
ох и попили мне крови эти "чек методы" в итоге... столкнулся с такой историей `Type definition error: [simple type, class org.hibernate.proxy.pojo.bytebuddy.ByteBuddyInterceptor]; nested exception is com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class org.hibernate.proxy.pojo.bytebuddy.ByteBuddyInterceptor and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS)` 
кое как вообще разобрался в чем была проблема)))

наверное по правильному вообще делать какой-то глобальный интерфейс проверок, от него имплементировать нужные классы  и переопределять методы где нужно, потому что нужны они почти во всех сервисах.

Reply:
конкретно эту проверку можно вообще не делать, а вообще простые проверки объекта делаются через костомные аннотации))